[v2.4.8 - Braintree Payments] Updated Braintree documentation for both Stores Purchase & Configuration Reference pages (#85)
AdobeDocs/commerce-admin.en: help/configuration-reference/sales/braintree.md

|[!UICONTROL Payment from Applicable Countries]|Website|Determines if you accept payments processed by PayPal through Braintree from all countries, or only specific countries. Options: `All Allowed Countries` / `Specific Countries`|
|[!UICONTROL Payment from Specific Countries]|Website|If applicable, identifies the specific countries from which you accept payments processed by Braintree.|
|[!UICONTROL Require Customer's Billing Address]| Website|Determines if the customer's billing address is required to submit an order. Options: `Yes` / `No`|
|[!UICONTROL Skip Order Review Step]|Website|Determines if customers are to be redirected to the review page before completing payment. Options: `Yes` / `No`|
|[!UICONTROL Debug]|Website|Determines if communications between the PayPal through Braintree system and your store are recorded in a log file. Options: `Yes` / `No`|
|[!UICONTROL Display on Shopping Cart]|Website|Determines if the PayPal button appears in the [mini cart](../../stores-purchase/cart-configuration.md#mini-cart) and on the [shopping cart](../../stores-purchase/cart.md) page. Options: `Yes` / `No`|

> |[!UICONTROL Send Package Tracking]|Website|Package tracking information will be sent to PayPal for PayPal transactions/orders only. Please enable the 'Send Cart Line Items for PayPal' configuration field in order for the 'Package Tracking' feature to work correctly. Options: `Yes` / `No`|

Reviewer: ```suggestion
|[!UICONTROL Send Package Tracking]|Website|Package tracking information will be sent to PayPal for PayPal transactions/orders only. You must enable the [!UICONTROL Send Cart Line Items for PayPal] configuration field in order for the [!UICONTROL Package Tracking] feature to work correctly. Options: `Yes` / `No`|
```
